packages/tools/lib/hbs2ui5/index.js

```diff
@@ -1,10 +1,11 @@
-const fs = require('fs');
+const fs = require('fs').promises;
 const getopts = require('getopts');
 const hbs2lit = require('../hbs2lit');
 const path = require('path');
 const litRenderer = require('./RenderTemplates/LitRenderer');
 const recursiveReadDir = require("recursive-readdir");
 const mkdirp = require('mkdirp');
+const fsExists = require("fs.promises.exists");
```

Comment:
I just double checked why there is no promise version, and the callback one is deprecated with the following explanation:
```
Using fs.exists() to check for the existence of a file before calling fs.open(), fs.readFile() or fs.writeFile()
is not recommended. Doing so introduces a race condition, since other processes may change the file's
state between the two calls. Instead, user code should open/read/write the file directly and handle the
error raised if the file does not exist.
```

I think we should drop the dependency and let mkdir always create and wrap the readFile in a try/catch